src/redux/slices/user/login.js

```diff
@@ -0,0 +1,47 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+
+const BASE_URL = import.meta.env.VITE_API_BASE_URL;
```

Comment:
@kirengaboris , no description, no screenshots, no label

Reply:
am still working on this its not yet ready that's why I haven't yet added the descriptions and the labels, I opened it so that I can be able to view it in vercel preview @denislohan